CHANGELOG.md

```
@@ -44,7 +44,8 @@ Other guiding principles:
 
 ### Changed
 
-- **amaru-ledger**: redesigned the `VolatileDB`, providing faster lookups and a design that is easier to reason about. ([#963][])]
+- **amaru-ledger**: redesigned the `VolatileDB`, providing faster lookups and a design that is easier to reason about. ([#963][])
+- **amaru-uplc**: replaced wildcard matches in `get_cost` so the compiler catches missing cost entries when new builtins are added. ([#971][])
```

markdownlint-cli2 (0.22.1)

[warning] Reference links and images should use a label that is defined
Missing link or image reference definition: "`#971`"

(MD052, reference-links-images)